_IBC/osmosis-intento.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "../ibc_data.schema.json",
+  "chain_1": {
+    "chain_name": "osmosis",
+    "client_id": "07-tendermint-3549",
+    "connection_id": "connection-10824"
+  },
+  "chain_2": {
+    "chain_name": "intento",
```

Comment:
intento must be chain 1 since it comes before alphabetically (same with the file name)